pkgs/top-level/all-packages.nix

```diff
@@ -21110,6 +21110,8 @@ with pkgs;
 
   tup = callPackage ../development/tools/build-managers/tup { };
 
+  truffleHog = pythonPackages.callPackage ../tools/security/truffleHog { };
```

Comment:
Can you keep the attribute entirely lowercase, and the same for the folder of the package.

Reply:
No, this should simply be `callPackage`, and the `trufflehog` expressions then has a parameter `python` instead of the individual Python packages and functions that are needed.

Reply:
Lowercased and swapped to `callPackage`; thanks!  Should I reword the commit message to `trufflehog: init at 2.0.91`, or leave it consistent with the pypi name?